lib/membrane_http_adaptive_stream/hls.ex

```diff
@@ -263,4 +260,26 @@ defmodule Membrane.HTTPAdaptiveStream.HLS do
   end
 
   defp serialize_ll_hls_tags(false, _target_partial_duration), do: ""
+
+  defp serialize_preload_hint_tag(true, track) do
+    # hls.js currently does not support EXT-X-PRELOAD_HINT tag
```

Comment:
This comment is irrelevant. Hls.js is just a javascript library with nothing to do with this code.